test/controllers/merge_status_controller_test.rb

```diff
@@ -33,5 +33,21 @@ class MergeStatusControllerTest < ActionController::TestCase
       assert_response :ok
       assert_predicate response.body, :blank?
     end
+
+    test "GET show prefers stacks with merge_queue_enabled" do
+      existing = shipit_stacks(:shipit)
+      Shipit::Stack.create(
+        repo_owner: existing.repo_owner,
+        repo_name: existing.repo_name,
+        environment: 'foo',
+        branch: existing.branch,
+        merge_queue_enabled: true,
+      )
+      existing.merge_queue_enabled = false
+      existing.save
```

Comment:
2 things:

  - You can do both using `update(merge_queue_enabled: false)`
  - In test suites I recommend to always use the bang versions `save! / update!`, this way you won't be tripped by a silent failure.